src/App.tsx

```diff
@@ -1,14 +1,27 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
```

Comment:
I don't think we need a state for this.

I think the optimal way will be to store `query` and `filterBy` states here in App.tsx, and make the `filteredTodos` a regular variable. Each change of `query` or `filterBy` will cause a re-render of App.tsx which means the variable `filteredTodos` will be re-evaluated on its own.